airbyte/mcp/__init__.py

```diff
@@ -67,14 +67,20 @@
 First install `uv` (`brew install uv`).
 
 Then, create a file named `server_config.json` (or the file name required by your MCP client)
-with the following content:
+with the following content. This uses `uvx` (from `brew install uv`) to run the MCP
+server. If a matching version Python is not yet installed, a `uv`-managed Python 
+version will be installed. This will also auto-update to use the "latest" Airbyte MCP release at time of launch.
```

Comment:
```suggestion
with the following content. This uses `uvx` (from `brew install uv`) to run the MCP
server. If a matching version Python is not yet installed, a `uv`-managed Python 
version will be installed automatically. This will also auto-update to use the
"latest" Airbyte MCP release at time of launch. You can alternatively pin to a 
specific version of Python and/or of the Airbyte library if you have special 
requirements.